### ISSUE TITLE
Publish to somewhere else besides jcenter

### ISSUE THREAD

**Issue by ink404** [realm/realm-android-adapters#165]
> **Feature Request**: Just fill in the first two sections below.

#### Goal

> Adapters should not be hosted on `jcenter`

#### Expected Results

> I'm able to include realm adapters in my project without using jcenter

#### Actual Results

>  I need to include the deprecated jcenter repo in my project

#### Version of Realm and tooling
Realm version(s): N/A

Android Studio version: N/A

Which Android version and device: N/A

**Comment by jk77** [comment]
This will become more of an issue soon since Android Studio is warning that 

> RepositoryHandler.jcenter() is scheduled to be removed in Gradle 9.0.
 
The changelog for Realm Java 10.4.0 (2021-03-26) says,

> All releases from 10.4.0 and forward are now found on mavenCentral() instead of jcenter().

but this great adapter is still in jcenter.

**Comment by savelyevdm** [comment]
> This will become more of an issue soon since Android Studio is warning that

It has become an issue, as project doesn't build anymore.

`Failed to resolve: io.realm:android-adapters:3.1.0`

**Comment by ThimonSistemans** [comment]
Yes this is an issue now. Please host alle versions in MavenCentral

**Comment by asadwaheed1** [comment]
yes unable to resolve now deleted everywhere

**Comment by adierebel** [comment]
> > This will become more of an issue soon since Android Studio is warning that
> 
> It has become an issue, as project doesn't build anymore.
> 
> `Failed to resolve: io.realm:android-adapters:3.1.0`

We got this issue too, the artifact is deleted from jcenter. See https://repo1.maven.org/maven2/io/realm/

**Comment by adierebel** [comment]
For those who are looking for solution, we recommend removing the dependencies in Gradle. Then embed these two files into your project https://github.com/realm/realm-android-adapters/tree/master/adapters/src/main/java/io/realm

I think this repository is abandoned, last updated 5 years ago. Maybe this is the reason the artifacts were removed.

**Comment by emilianocds** [comment]
I removed it from gradle, added the .aar to the libs folder in my project, and added `implementation file ("libs/...")` code to gradle

**Comment by didierboka** [comment]
@emilianocds can you share the .aar file please ?

**Comment by nightcap79** [comment]
> [@emilianocds](https://github.com/emilianocds) can you share the .aar file please ?

you can find the files [here](https://maven.joyyinc.com/nexus/content/repositories/public/io/realm/android-adapters/3.1.0/)

**Comment by nightcap79** [comment]
Add this repo to your  allprojects  repositories
        `maven { url "https://maven.joyyinc.com/nexus/content/repositories/public" }`
will solve your problem
